README.md

```diff
@@ -13,19 +13,19 @@ The service are using authentication, not everything is accessible trough the DB
 
 ## CRUD Operations:
 
-All requests are send to: `http://localhost:3030`: /event/, /user/, /organization/. Supported requests are `GET`, `POST`, `PUT`!
+All requests are send to: `http://localhost:3030`: /event/, /user/. Supported requests are `GET`, `POST`, `PUT`!
 
 ## Authentication:
 
-The service is not initialized with any data. You need to create it by yourself. To register an event you must register yourself as `organization`! If you register as `user`, you wil only able to read data and edit your profile.
+The service is not initialized with any data. You need to create it by yourself. To register an event you must register yourself as `user` with `role`: `organizer`! If you register as `role`: `regular`, you wil only able to read data and edit your profile.
 
 ### Register:
 
-Create a new `user` by sending a POST request to `http://localhost:3030`: + `/user/registerUser` with properties: `email` and `password`. The service automatically creates a session and returns object with `accessToken` inside, that can be used for authorized requests. You can also register `Organization` by sending a POST request to `http://localhost:3030:` + `/organization/registerOrganization` with properties: `name`, `email`, `managerFirstName`, `managerLastName`, `phone`, `region`, `password`. The service automatically creates a session and returns object with `accessToken` inside, that can be used for authorized requests.
+Create a new `user` by sending a POST request to `http://localhost:3030`: + `/user/registerUser` with properties: `email`, `password`, `repass` and `role`, if the role is `regular`. If the `role` is `organizer` you need to pass `email`, `password`, `repass`, `role`, `organizationName`, `firstName`, `lastName`, `region`, `phone`. The service automatically creates a session and returns object with `accessToken` inside, that can be used for authorized requests.
```

Comment:
Why "and `role`, if the role is `regular`" ? Isn't it by default without need of passing it? 
Also, isn't it `organizatorName` , not `organizationName` ? Double check the properties.